src/frontend/components/property-type/richtext/show.tsx

```diff
@@ -1,31 +1,25 @@
-import React, { ReactNode } from 'react'
-import { ValueGroup, Text, Box } from '@adminjs/design-system'
-
+import { Box, Text, ValueGroup } from '@adminjs/design-system'
+import React, { FC, useEffect, useRef } from 'react'
 import { EditPropertyProps } from '../base-property-props'
 
-export default class Show extends React.PureComponent<EditPropertyProps> {
-  private contentRef: React.RefObject<any>
-
-  constructor(props: EditPropertyProps) {
-    super(props)
-    this.contentRef = React.createRef()
-  }
+const Show: FC<EditPropertyProps> = (props) => {
+  const { property, record } = props
+  const contentRef = useRef<any>(null)
 
-  componentDidMount(): void {
-    const { property, record } = this.props
-    const value = record.params[property.path]
-    this.contentRef.current.innerHTML = value
-  }
+  useEffect(() => {
+    if (contentRef.current) {
+      const value: string = record.params[property.path] || ''
+      contentRef.current.innerHTML = value
+    }
+  }, [])
```

Comment:
Use `dangerouslySetInnerHTML` instead. Would be nice to pass it through some anti-xss tool like [`xss`](https://www.npmjs.com/package/xss).